README.md

```diff
@@ -21,14 +21,26 @@ sudo make install
 make clean
 ```
 
+### Uninstalling 
+
+```
+cd /path/to/xwinwrap
+make
+sudo make uninstall
+make clean
```

Comment:
why are you making clean if you're about to delete the entire folder

Reply:
You are right, this line can be deleted. My fault.